packages/big-design/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,74 @@
+import { ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { useState } from 'react';
+
+import { useUniqueId } from '../../hooks';
+import { typedMemo } from '../../utils';
+import { Box } from '../Box';
+
+import { StyledButton, StyledIconWrapper, StyledTitle } from './styled';
+
+export interface CollapseProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
+  title: string;
+  initiallyOpen?: boolean;
+  onChange?(isOpen: boolean): void;
+}
+
+export const Collapse: React.FC<CollapseProps> = typedMemo(
+  ({ children, title, onChange, initiallyOpen = false, ...props }) => {
+    const [isOpen, setIsOpen] = useState(initiallyOpen);
+    const uniqueCollapseId = props.id ? props.id : useUniqueId('collapse');
+    const triggerId = `${uniqueCollapseId}-title`;
+    const panelId = `${uniqueCollapseId}-panel`;
+
+    const handleTitleClick = () => {
+      setIsOpen(!isOpen);
```

Comment:
I believe it's safer with hooks not to pass in the current state and use a function instead: 
```suggestion
      setIsOpen(prevIsOpen => !prevIsOpen);
```

Reply:
Thanks, Jordan. But I used version which Chancellor proposed.

```const nextIsOpen = !isOpen;
      setIsOpen(nextIsOpen);```